lib/tjbot.js

```diff
@@ -51,14 +50,7 @@ function TJBot(hardware, configuration, credentials) {
     }
 
     // import configuration params
-    this.configuration = {};
-    TJBot.prototype.configurationParameters.forEach(function(param) {
-        if (configuration != undefined && configuration[param] != undefined) {
-            this.configuration[param] = configuration[param];
-        } else {
-            this.configuration[param] = TJBot.prototype.defaultConfiguration[param];
-        }
-    }, this);
+    this.config = configuration;
```

Comment:
For me `configuration` or `config` are good.
I just choose `config` to reduce length of some variable names, example (config.js): 
```
exports.config.conversationWorkspaceId = '7a8a08f9-4543-4b0a-a8f4-53a0fd306cdf';
exports.config.visionTextConfidenceThreshold = 0.1;
```

I used the same variable name in all parts of app. If it is `config` in config.js it is also `config` in tjbot.js

Should I change all to `configuration`?
```
exports.configuration.conversationWorkspaceId = '7a8a08f9-4543-4b0a-a8f4-53a0fd306cdf';
exports.configuration.visionTextConfidenceThreshold = 0.1;
```